main.tf

```diff
@@ -122,6 +158,123 @@ resource "observe_dataset" "ingress_logs" {
   }
 }
 
+resource "observe_dataset" "nginx_metrics" {
```

Comment:
Just `metrics`. Move these new additions to a `metrics.tf`.

Reply:
Changed